### PR TITLE
Add Tracks event to ccpa opt-out click

### PR DESCRIPTION
**Pull request by robertsreberski**
#### Proposed Changes

* Add `a8c_ccpa_optout` to exceptions that do not use `calypso_` prefix
* Implement `a8c_ccpa_optout` on CCPA opt-out

#### Testing Instructions

* Checkout this PR and run `yarn start`
* Enter `http://calypso.localhost:3000/me/privacy?flags=cookie-banner,ad-tracking`
* Ensure you've set cookies `region=Colorado`, `country_code=US`
* Refresh the page, you should see CCPA setting
* Click on toggle to opt-out from advertising
* You should get Tracks event `a8c_ccpa_optout`

#### Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

Related to #

### CODE REVIEW

**Review by gmovr — APPROVED** [review]
LGTM, I re-ran the e2e, just a flaky test. 🚢

File: client/lib/analytics/utils/use-do-not-sell.ts [hunk 1]
```diff
@@ -33,6 +34,15 @@ export default () => {
 		( isActive: boolean ) => {
 			// isActive = true means user has opted out of "advertising" tracking
 			const prefs = setTrackingPrefs( { ok: true, buckets: { advertising: ! isActive } } );
+
+			if ( ! isActive ) {
```
Comment (CodeyGuyDylan) [inline client/lib/analytics/utils/use-do-not-sell.ts]:
```suggestion
			if ( isActive ) {
```
Correct me if I am wrong, but based on the name `a8c_ccpa_optout` and the description of this PR, we want this event to trigger when somebody "opt's out" by turning **on** the "Do Not Sell or Share My Data" toggle correct?

If so, I think this change should be made, I am only getting the tracks even triggering when I turn the toggle off

**Review by CodeyGuyDylan — CHANGES_REQUESTED** [review]
Unless I am misunderstanding the goal here, I believe the event is being triggered incorrectly

**Review by CodeyGuyDylan — APPROVED** [review]
Tests well and LGTM :shipit:

**Pull request merged**